### PR TITLE
Update partitions limitations

### PR DESCRIPTION
**Pull request by sukhhanda1**
We have a limitation of 2048 characters on partition rule length. Updating the same in the documentation. More Details - https://sumologic.slack.com/archives/C0P4K20RL/p1695810197382599

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/manage/partitions-data-tiers/index.md [hunk 1]
```diff
@@ -35,7 +35,8 @@ You define the data that will reside in a partition by defining a routing expres
    * You can’t change or reuse a partition name.
    * You can’t change the data tier the partition resides in.
 * Partitions cannot be deleted, although you can [decommission](decommission-partition.md) them. This is because a partition may include log messages that aren’t stored anywhere else, so if it’s deleted, messages will be lost. If you no longer need a partition, you can decommission it.
-* Partition names cannot start with `sumologic_` or an underscore `_`. 
+* Partition names cannot start with `sumologic_` or an underscore `_`.
+* Partition Routing rule length cannot exceed 2048 characters.
```
Comment (jpipkin1) [inline docs/manage/partitions-data-tiers/index.md:39]:
```suggestion
* Partition routing rule length cannot exceed 2048 characters.
```

**Review by jpipkin1 — APPROVED** [review]
Your change looks good, @sukhhanda1. Approved.

**Comment by jpipkin1** [comment]
@sukhhanda1, we can merge your doc update as soon as PR #3123 is done adding your name to our [.clabot](https://github.com/SumoLogic/sumologic-documentation/blob/main/.clabot) file.

**Comment by jpipkin1** [comment]
@kimsauce, the clabot verification step didn't happen for this PR. I set this PR to auto-merge, and the PR went ahead and merged even though PR https://github.com/SumoLogic/sumologic-documentation/pull/3123 hadn't added the contributor to the clabot file yet.

**Comment by kimsauce** [comment]
> @kimsauce, the clabot verification step didn't happen for this PR. I set this PR to auto-merge, and the PR went ahead and merged even though PR #3123 hadn't added the contributor to the clabot file yet.

thanks for the heads up - i'll look into it

**Pull request merged**